app/src/components/common/outcome_table/index.tsx

```diff
@@ -159,15 +159,12 @@ export const OutcomeTable = (props: Props) => {
     )
   }
 
-  const yesRow = renderTableRow(balance[0], pricesAfterTrade && pricesAfterTrade[0])
-  const noRow = renderTableRow(balance[1], pricesAfterTrade && pricesAfterTrade[1])
-
-  let rows = [yesRow, noRow]
-  if (withWinningOutcome) {
-    if (balance[1].winningOutcome) {
-      rows = [noRow, yesRow]
-    }
-  }
-
-  return <TableStyled head={renderTableHeader()}>{rows}</TableStyled>
+  const renderTable = () =>
+    balance
+      .sort((a, b) => (a.winningOutcome === b.winningOutcome ? 0 : a.winningOutcome ? -1 : 1))
```

Comment:
This sort exist because we need to order the array by winningOutcome, is a boolean

Reply:
Got it, let's add a comment saying something like `Put winning outcome first`.

Reply:
Btw, an alternative way of doing this is to use [`_.partition`](https://lodash.com/docs/4.17.15#partition):

```typescript
const [winningOutcomes, losingOutcomes] = _.partition(balances, 'winningOutcome')
if (winningOutcomes.length > 1) {
  logger.warning('There should be only one winning outcome')
}
const sortedOutcomes = [...winningOutcomes, ...losingOutcomes]
```

Reply:
@fvictorio I will add the comment, regarding lodash, we need to install it and maybe is an overkill, what do you think ?

Reply:
It's OK the way it is.